src/Components/Web.JS/src/Platform/BootConfig.ts

```diff
@@ -37,3 +38,9 @@ export interface ResourceGroups {
 }
 
 export type ResourceList = { [name: string]: string };
+
+export enum ICUDataMode {
+  Default,
+  All,
+  Invariant
+}
```

Comment:
On naming, do you think `Sharded` might be clearer than `Default`? Sometimes "default" means "one of the other values but I'm not specifying which" whereas here it's a different behavior than either of the other two.